bindings/java/src/org/sleuthkit/datamodel/Host.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Sleuth Kit Data Model
+ *
+ * Copyright 2021-2021 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel;
+
+import java.util.Objects;
+
+/**
+ *
+ * Encapsulates a host.
+ */
+public class Host {
+
+	private final long id;
+	private String name;
+	private HostStatus status;
+
+	
+
+	Host(long id, String name) {
+		this(id, name, HostStatus.ACTIVE);
+	}
+
+	Host(long id, String name, HostStatus status) {
+		this.id = id;
+		this.name = name;
+		this.status = status;
+	}
+
+	public long getId() {
```

Comment:
Missing method docs on public methods that are not overrides. I think we should have something for the web page.